hasjob/models/domain.py

```diff
@@ -104,6 +105,11 @@ def get(cls, name, create=False):
             db.session.add(result)
         return result
 
+    @classmethod
+    def autocomplete(cls, prefix):
+        search = make_name(prefix) + '%'
+        return cls.query.filter(cls.name.like(search), cls.is_banned == False).all()  # NOQA
```

Comment:
Maybe use `ilike` for lowercase matching, although `ilike` does have a performance impact (for good performance, search by `lower(name)` with an index on that; see Lastuser's User.autocomplete for how).